{"date":"Mon May 23 2022 18:09:24 GMT-0500 (hora estándar de Perú)","error":{},"exception":true,"level":"error","message":"uncaughtException: createError is not defined\nReferenceError: createError is not defined\n    at Object.Execute (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\application\\use-cases\\auth\\Authenticate.js:5:13)\n    at authController (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\controllers\\auth-controller.js:10:27)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","os":{"loadavg":[0,0,0],"uptime":884135},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\app.js"],"cwd":"D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18298299,"external":19351190,"heapTotal":21770240,"heapUsed":20310360,"rss":51355648},"pid":3808,"uid":null,"version":"v16.14.0"},"stack":"ReferenceError: createError is not defined\n    at Object.Execute (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\application\\use-cases\\auth\\Authenticate.js:5:13)\n    at authController (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\controllers\\auth-controller.js:10:27)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","trace":[{"column":13,"file":"D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\application\\use-cases\\auth\\Authenticate.js","function":"Object.Execute","line":5,"method":"Execute","native":false},{"column":27,"file":"D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\controllers\\auth-controller.js","function":"authController","line":10,"method":null,"native":false},{"column":5,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":96,"method":null,"native":false}]}
{"date":"Mon May 23 2022 18:11:00 GMT-0500 (hora estándar de Perú)","error":{},"exception":true,"level":"error","message":"uncaughtException: createError is not defined\nReferenceError: createError is not defined\n    at Object.Execute (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\application\\use-cases\\auth\\Authenticate.js:5:13)\n    at authController (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\controllers\\auth-controller.js:10:27)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","os":{"loadavg":[0,0,0],"uptime":884232},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\app.js"],"cwd":"D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18421299,"external":23402901,"heapTotal":36188160,"heapUsed":22060096,"rss":65863680},"pid":7748,"uid":null,"version":"v16.14.0"},"stack":"ReferenceError: createError is not defined\n    at Object.Execute (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\application\\use-cases\\auth\\Authenticate.js:5:13)\n    at authController (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\controllers\\auth-controller.js:10:27)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","trace":[{"column":13,"file":"D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\application\\use-cases\\auth\\Authenticate.js","function":"Object.Execute","line":5,"method":"Execute","native":false},{"column":27,"file":"D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\controllers\\auth-controller.js","function":"authController","line":10,"method":null,"native":false},{"column":5,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":96,"method":null,"native":false}]}
{"date":"Mon May 23 2022 18:11:15 GMT-0500 (hora estándar de Perú)","error":{},"exception":true,"level":"error","message":"uncaughtException: createError is not defined\nReferenceError: createError is not defined\n    at Object.Execute (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\application\\use-cases\\auth\\Authenticate.js:5:13)\n    at authController (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\controllers\\auth-controller.js:10:27)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","os":{"loadavg":[0,0,0],"uptime":884247},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\app.js"],"cwd":"D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18421299,"external":23402901,"heapTotal":36188160,"heapUsed":22130480,"rss":65740800},"pid":792,"uid":null,"version":"v16.14.0"},"stack":"ReferenceError: createError is not defined\n    at Object.Execute (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\application\\use-cases\\auth\\Authenticate.js:5:13)\n    at authController (D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\controllers\\auth-controller.js:10:27)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","trace":[{"column":13,"file":"D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\application\\use-cases\\auth\\Authenticate.js","function":"Object.Execute","line":5,"method":"Execute","native":false},{"column":27,"file":"D:\\Proyecto\\Proyecto_Web\\NodeJs\\LoanApp\\Backend\\src\\controllers\\auth-controller.js","function":"authController","line":10,"method":null,"native":false},{"column":5,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":96,"method":null,"native":false}]}
{"level":"error","message":"422 - Validation error - /api/auth - POST - ::1"}
{"level":"error","message":"422 - Validation error - /api/auth - POST - ::1"}
{"level":"error","message":"422 - Validation error - /api/auth - POST - ::1"}
{"level":"error","message":"404 - Not found - /api/auth - POST - ::1"}
{"level":"error","message":"404 - Not found - /api/auth - POST - ::1"}
{"level":"error","message":"404 - The email or password dosen't exists - /api/auth - POST - ::1"}
{"date":"Wed May 25 2022 18:04:49 GMT-0500 (Peru Standard Time)","error":{},"exception":true,"level":"error","message":"uncaughtException: The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.\nMongooseError: The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.\n    at NativeConnection.Connection.openUri (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/connection.js:689:11)\n    at /Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/index.js:342:10\n    at /Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/index.js:1181:10)\n    at Mongoose.connect (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/index.js:341:20)\n    at MongodbDatabase.getConnection (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/application/contracts/MongoDBDatabaseService.js:7:25)\n    at InMemoryDatabaseServices.initDatabase (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/frameworks/persistance/InMemoryDatabaseServices.js:14:38)\n    at Object.<anonymous> (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/app.js:13:37)","os":{"loadavg":[1.40283203125,1.71044921875,4.9130859375],"uptime":2992},"process":{"argv":["/usr/local/bin/node","/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/app.js"],"cwd":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs","execPath":"/usr/local/bin/node","gid":20,"memoryUsage":{"arrayBuffers":18882662,"external":23333484,"heapTotal":34193408,"heapUsed":23840248,"rss":70418432},"pid":3693,"uid":501,"version":"v16.14.0"},"stack":"MongooseError: The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.\n    at NativeConnection.Connection.openUri (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/connection.js:689:11)\n    at /Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/index.js:342:10\n    at /Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/index.js:1181:10)\n    at Mongoose.connect (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/index.js:341:20)\n    at MongodbDatabase.getConnection (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/application/contracts/MongoDBDatabaseService.js:7:25)\n    at InMemoryDatabaseServices.initDatabase (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/frameworks/persistance/InMemoryDatabaseServices.js:14:38)\n    at Object.<anonymous> (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/app.js:13:37)","trace":[{"column":11,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/connection.js","function":"NativeConnection.Connection.openUri","line":689,"method":"openUri","native":false},{"column":10,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/index.js","function":null,"line":342,"method":null,"native":false},{"column":5,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/helpers/promiseOrCallback.js","function":null,"line":32,"method":null,"native":false},{"column":null,"file":null,"function":"new Promise","line":null,"method":null,"native":false},{"column":10,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/helpers/promiseOrCallback.js","function":"promiseOrCallback","line":31,"method":null,"native":false},{"column":10,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/index.js","function":"Mongoose._promiseOrCallback","line":1181,"method":"_promiseOrCallback","native":false},{"column":20,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/mongoose/lib/index.js","function":"Mongoose.connect","line":341,"method":"connect","native":false},{"column":25,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/application/contracts/MongoDBDatabaseService.js","function":"MongodbDatabase.getConnection","line":7,"method":"getConnection","native":false},{"column":38,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/frameworks/persistance/InMemoryDatabaseServices.js","function":"InMemoryDatabaseServices.initDatabase","line":14,"method":"initDatabase","native":false},{"column":37,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/app.js","function":null,"line":13,"method":null,"native":false}]}
{"date":"Wed May 25 2022 18:04:49 GMT-0500 (Peru Standard Time)","error":{},"exception":true,"level":"error","message":"uncaughtException: Route.get() requires a callback function but got a [object Undefined]\nError: Route.get() requires a callback function but got a [object Undefined]\n    at Route.<computed> [as get] (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/express/lib/router/route.js:211:15)\n    at Function.proto.<computed> [as get] (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/express/lib/router/index.js:521:19)\n    at creditorRoutes (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/frameworks/routes/creditor.routes.js:9:12)\n    at apiRoutes (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/frameworks/routes/index.js:11:29)\n    at /Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/app.js:18:21\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","os":{"loadavg":[1.40283203125,1.71044921875,4.9130859375],"uptime":2992},"process":{"argv":["/usr/local/bin/node","/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/app.js"],"cwd":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs","execPath":"/usr/local/bin/node","gid":20,"memoryUsage":{"arrayBuffers":18888051,"external":23338913,"heapTotal":34193408,"heapUsed":23908504,"rss":70582272},"pid":3693,"uid":501,"version":"v16.14.0"},"stack":"Error: Route.get() requires a callback function but got a [object Undefined]\n    at Route.<computed> [as get] (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/express/lib/router/route.js:211:15)\n    at Function.proto.<computed> [as get] (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/express/lib/router/index.js:521:19)\n    at creditorRoutes (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/frameworks/routes/creditor.routes.js:9:12)\n    at apiRoutes (/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/frameworks/routes/index.js:11:29)\n    at /Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/app.js:18:21\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","trace":[{"column":15,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/express/lib/router/route.js","function":"Route.<computed> [as get]","line":211,"method":"<computed> [as get]","native":false},{"column":19,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/node_modules/express/lib/router/index.js","function":"Function.proto.<computed> [as get]","line":521,"method":"<computed> [as get]","native":false},{"column":12,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/frameworks/routes/creditor.routes.js","function":"creditorRoutes","line":9,"method":null,"native":false},{"column":29,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/frameworks/routes/index.js","function":"apiRoutes","line":11,"method":null,"native":false},{"column":21,"file":"/Users/phantamium/Documents/Node js/LoanApp-Nodejs/src/app.js","function":null,"line":18,"method":null,"native":false},{"column":5,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":96,"method":null,"native":false}]}
